src/functions.c

```diff
@@ -411,6 +425,20 @@ void functionStatsCommand(client *c) {
     dictReleaseIterator(iter);
 }
 
+static void functionListReplyFlags(client *c, functionInfo *fi) {
+    int flagcount = 0;
+    void *flaglen = addReplyDeferredLen(c);
+
+    for (functionFlag *flag = function_flags_def; flag->str ; ++flag) {
+        if (fi->f_flags & flag->flag) {
+            addReplyStatus(c, flag->str);
+            ++flagcount;
+        }
+    }
+
+    setDeferredSetLen(c, flaglen, flagcount);
```

Comment:
please avoid using a nested deferred reply. see what i did in `addReplyCommandFlags`

Reply:
Fixed